test/specs/graphql/crud_federation_test.js

```diff
@@ -0,0 +1,195 @@
+import { expect } from 'chai';
+import { apis_page } from '../../../lib/pom/Apis_page';
+import { login_page } from '../../../lib/pom/Login_page';
+import { main_page } from '../../../lib/pom/Main_page';
+import { runFederationExample } from '../../../lib/utils/utils';
+import { API_connection } from '../../../lib/utils/api_connections/API_connection';
+
+describe('CRUD simple GraphQL (proxy-only) API', () => {
+
+    let localFederationUrl;
+
+    if(process.env.RUN_ENV == "CI") localFederationUrl = "172.17.0.1";
+    else localFederationUrl = "localhost";
+
+    const apiDetails = {
+        supergraphName: "Super-test",
+        usersSubgraphUrl: `http://${localFederationUrl}:4001/query`,
+        usersSubgraphName: "Users-test",
+        productsSubgraphUrl: `http://${localFederationUrl}:4002/query`,
+        productsSubgraphName: "Products-test",
+        reviewsSubgraphUrl: `http://${localFederationUrl}:4003/query`,
+        reviewsSubgraphName: "Reviews-test"        
+    };
+
+    const federationConnectionRetries = 5;
+    const federationExampleTestUrl = "http://localhost:4000/"
+    const api_connection = new API_connection(federationExampleTestUrl);
+    const expectedFederationResponse = { data: { me: { id: '1234' } } }
+
+    const federationTestRequestConfig = {
+        path: "query",
+        body: `{"query":"query{me{id}}","variables":{}}`
+    }    
+
+    let $supergraphTableElement;
```

Comment:
just to confirm: is this table not working with our wrapper?
If it works, I think we could reuse the methods from there to click/check cells

Reply:
I'm not sure why you ask. I'm using our wrapper. Dropdown object to be precise.